src/lib/features/dependent-features/dependent-features-service.ts

```diff
@@ -2,6 +2,7 @@ import { InvalidOperationError } from '../../error';
 import { CreateDependentFeatureSchema } from '../../openapi';
 import { IDependentFeaturesStore } from './dependent-features-store-type';
 
+export type FeatureDependencyId = { parent: string; child: string };
```

Comment:
Isn't this a circular dependency? dependent-features-store-type ends up depending on dependent-feature-service, and dependent-feature-service depends on dependent-features-store-type. Bringing in the dependent-features-store causes another circle since dependent-features-service calls out to the store, but the store depends on the service?

Reply:
solved